hw/BUILD

```diff
@@ -17,6 +17,37 @@ fusesoc_build(
     data = ["//hw/ip/otbn:all_files"],
     systems = ["lowrisc:dv:chip_verilator_sim"],
     target = "sim",
+    visibility = ["//visibility:private"],
+)
+
+# This is used in CI steps that do not want to run Verilator tests, and thus
+# do not want to accidentally build Verilator. This causes the Verilator
```

Comment:
Probably worth getting the terminology right here: Verilator is a program that we depend on. We use it to build a verilated model of the chip.